src/compile-middleware.ts

```diff
@@ -123,6 +123,11 @@ export function babelCompile(
       // TODO(justinfagnani): Use path-is-inside, but test on Windows
       const isRootPathRequest = request.path.startsWith(`/${packageName}`);
       const isComponentRequest = request.baseUrl === `/${componentUrl}`;
+      let componentRequestRelationship: undefined|'../../'|'../';
+      if (isComponentRequest) {
+        const isScopedComponent = packageName.startsWith('@');
```

Comment:
related to the comment in polymer-build, if you just pass the package name, you shouldn't have to rely on special characters or an assumption that there's only one or two levels of scopes.